crates/core/src/operations/optimize.rs

```diff
@@ -238,6 +400,8 @@ impl<'a> OptimizeBuilder<'a> {
             max_concurrent_tasks: num_cpus::get(),
             max_spill_size: 20 * 1024 * 1024 * 1024, // 20 GB.
             optimize_type: OptimizeType::Compact,
+            sort_enabled: true,
+            sort_columns: vec!["objectId".to_string(), "dateTime".to_string()],
```

Comment:
These seem like they are specific to your data? I don't think they should be the default.